meson.build

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 Endless Mobile, Inc.
+
+project('eos-knowledge-renderer', 'c', version: '0.0.0', license: 'LGPL2.1+',
+    meson_version: '>= 0.42.0',
+    default_options: ['c_std=c99', 'warning_level=3'])
+
+gnome = import('gnome')
+pkg = import('pkgconfig')
+
+version_components = meson.project_version().split('.')
+api_version = version_components[0]
+api_name = '@0@-@1@'.format(meson.project_name(), api_version)
+libtool_version = '0.0.0'
+namespace_name = 'Eknr'
+
+# Build options
+
+mathjax_dir = get_option('mathjax_dir')
+
+# Dependencies
+
+libendless_req = '>= 0.5'
+
+gio = dependency('gio-2.0')
+glib = dependency('glib-2.0')
+gobject = dependency('gobject-2.0')
+libendless = dependency('endless-0', version: libendless_req)
+json_glib = dependency('json-glib-1.0')
+mustache = dependency('mustache_c-1.0')
+
+# Data files
+
+subdir('data')
+
+# Resources
+
+gresources = gnome.compile_resources(
+    'eos-knowledge-renderer-resources',
+    'eos-knowledge-renderer.gresource.xml',
+    dependencies: scss_generated_targets,
+    source_dir: [meson.current_source_dir(), meson.current_build_dir()],
+    c_name: 'eknr'
+)
+
+subdir('eknrenderer')
+
+# Generated Files
+
+config = configuration_data()
+config.set_quoted('EKNR_VERSION', meson.project_version())
+config.set_quoted('MATHJAX_PATH', mathjax_dir)
+configure_file(configuration: config, output: 'config.h')
+
+requires = ['glib-2.0', 'gio-2.0', 'gobject-2.0']
+requires_private = [
+    'endless-0 @0@'.format(libendless_req),
+    'json-glib-1.0'
+]
+pkg.generate(filebase: api_name, libraries: [main_library],
+    description: 'Software development kit for applications for the developing world, (legacy content renderer)',
```

Comment:
Not a fan of this description. This library really has nothing to do with the developing world...

Reply:
Neither - it was copy-pasted from dmodel :)

Reply:
And I copy-pasted it from ekncontent...